modules/core/src/shaderlib/project/constants.js

```diff
@@ -0,0 +1,9 @@
+export const PROJECT_COORDINATE_SYSTEM = {
+  LNG_LAT: 1.0,
+  LNGLAT_AUTO_OFFSET: 4.0,
```

Comment:
1) What is `AUTO_OFFSET`? Add a comment?

2) While it makes some sense not to change the numeric values in the public constants, the internal values can be rearranged to match the order of declarations

Reply:
That is the new mode. (experimental)